### PR TITLE
PLAT-1605: Add title and hyperlink to tooltip

### PR DESCRIPTION
**Pull request by mihaelpalfi**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1605)

#### Description
As per JIRA ticket

image

#### ChangeLOG

[Changelog](https://zen-ui.zengrc.com/?path=/docs/changelog--page)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

**Review by homer0 — APPROVED** [review]
🤘
image

It seems that it has the same "issue" as the avatar detail (shorter body makes the title multiline), but that can taken on a different ticket/PR, as the scope of this one is already covered.

File: src/components/zen-tooltip/zen-tooltip.scss [hunk 1]
```diff
@@ -11,3 +11,22 @@ $arrow_size: 0.3rem;
...
+
+.header {
+  padding-bottom: 0.5rem;
+}
+
+.link {
+  display: flex;
+  color: $color-blue-700;
+  text-decoration: none;
+  cursor: grab;
```
Comment (chavlji) [inline src/components/zen-tooltip/zen-tooltip.scss:23]:
```suggestion
  cursor: pointer;
```
😂

File: src/components/zen-tooltip/zen-tooltip.stories.mdx [hunk 2]
```diff
@@ -126,6 +149,14 @@ Limit tooltip height by setting `max-height` ('4rem' in example below).
   <Story name="scrollable">{Scrollable.bind({})}</Story>
 </Canvas>
 
+#### 4. Tooltip with title and hyperlink:
+
+Set property header and link.
```
Comment (chavlji) [inline src/components/zen-tooltip/zen-tooltip.stories.mdx:154]:
```suggestion
Set property `header` and/or `link`.
```

**Review by chavlji — CHANGES_REQUESTED** [review]
Very nice and clean!

File: src/components/zen-tooltip/zen-tooltip.scss (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-tooltip/zen-tooltip.scss:23]: uppss. Changed.

File: src/components.d.ts [hunk 3]
```diff
@@ -973,10 +973,22 @@ export namespace Components {
           * Show and hide delay. Eg. '100' - both show & hide 100ms. '100 500' - show 100ms, hide 500ms.
          */
         "delay": string;
+        /**
+          * Hyperlink title
+         */
+        "header": string;
```
[the remark this answers is not in the document]
Comment (mihaelpalfi) [reply src/components.d.ts:979]: Renamed.

File: src/components/zen-tooltip/zen-tooltip.stories.mdx (hunk 2 above)
Comment (mihaelpalfi) [reply src/components/zen-tooltip/zen-tooltip.stories.mdx:154]: Done

**Comment by mihaelpalfi** [comment]
> 🤘
> image
> 
> It seems that it has the same "issue" as the avatar detail (shorter body makes the title multiline), but that can taken on a different ticket/PR, as the scope of this one is already covered.

Shouldn't it go to new line?

**Review by chavlji — APPROVED** [review]

**Pull request merged**